lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1657,6 +1752,25 @@ void hlsl::SerializeDxilContainerForModule(
   if (pModule->GetShaderModel()->IsLib()) {
     DXASSERT(pModule->GetSerializedRootSignature().empty(),
              "otherwise, library has root signature outside subobject definitions");
+    // Write the DxilCompilerVersion (VERS) part.        
+    unsigned ValMajor, ValMinor;
+    pModule->GetValidatorVersion(ValMajor, ValMinor);
+    bool bValidatorAtLeast_1_8 = DXIL::CompareVersions(ValMajor, ValMinor, 1, 8) >= 0;
```

Comment:
We already have `ValMajor` and `ValMinor` retrieved earlier, and used to initialize some bool variables there.  Can we just move the `bValidatorAtLeast_1_8` up with the others and remove the extra validator version retrieval here?